specs/specs.go

```diff
@@ -36,3 +36,24 @@ func (projects *ProjectsSpec) LoadProjects(path string) error {
 	projects.RootDirectory = os.ExpandEnv(projects.RootDirectory)
 	return nil
 }
+
+// Write write projects to the given path
+func (projects *ProjectsSpec) Write(path string) error {
+	jsonData, err := json.MarshalIndent(projects, "", "  ")
+	if err != nil {
+		return err
+	}
+
+	writeErr := ioutil.WriteFile(path, jsonData, 0644)
```

Comment:
Because `LoadProjects` sets `projects.RootDirectory` with `os.ExpandEnv` this write here will change the repo config. I'm not sure if that is desirable.

For example, if my config had: `"rootDirectory": "${HOME}/projects"`
It would be written out as `"rootDirectory": "/Users/charliejones/projects"`

Reply:
Oh like maybe you want the file to be portable?

Reply:
Yeah, that's what I'm thinking. I frequently hop between Linux and Mac, and I sync my configs, so this could be problematic for me.

I suppose this could be handled at a later time though.

Reply:
I think I'd rather fix this in its own PR